widgets/Salesforce-connector.json

```diff
@@ -59,6 +59,15 @@
           "widget-attributes" : {
             "default": "https://login.salesforce.com/services/oauth2/token"
           }
+        },
+        {
+          "widget-type": "number",
```

Comment:
Is this json for connection manager?

Reply:
yes, we have moved connectTimeout to connector config as it is specific to connection properties